### PR TITLE
Style Guide and Diataxis Review: Identity I

### PR DESCRIPTION
**Pull request by charlesthompson3**
# Description of change
TW style guide and diataxis review applied to Identity documentation. Sections reviewed include:

- Introduction (overall)
- Decentralized Identity
- Getting Started - Overview

## Type of change

- [x] Documentation Fix

## Change checklist

- [x] I have followed the contribution guidelines for this project
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by charlesthompson3** [comment]
@lucas-tortora @aleksei-korolev

**Comment by charlesthompson3** [comment]
@lucas-tortora @aleksei-korolev

File: documentation/docs/decentralized_identity.md [hunk 1]
```diff
@@ -1,99 +1,103 @@
...
 
-Self-Sovereign Identity is about returning autonomy and privacy to the individual, while also improving our online experience. Some movements focus on data privacy, preventing companies from using our information all together, but the IOTA Identity allows controlled information flow. The user can create a single online profile, containing all our personal information. They can decide who they share what information with, and a Verifier is able to verify the information to be correct, making the data trustworthy. This moves their online profile from a statistical estimation by corporate entities to an accurate and verifiable profile under their own control.
+Self-Sovereign Identity is altogether, but the IOTA Identity allows for a controlled flow of information. The user can create a single online profile containing all our personal information. They can decide who they share what information with, and a Verifier verifies if the information is correct, making the data trustworthy. This moves their online profile from a statistical estimation by corporate entities to an accurate and verifiable profile under their own control.
```
Comment (aleksei-korolev) [inline documentation/docs/decentralized_identity.md:34]: I like the idea, but I think that "Self-Sovereign Identity is altogether" is a fair bit confusing. "SSI contains all information, but you control which part of it you want to reveal with IOTA DID" or along the lines?

BTW I really like that Jelle is a loudmouth. It does help to understand why DIDs are so cool and important and all.

Comment (aleksei-korolev) [reply documentation/docs/decentralized_identity.md:34]: > Verifier verifies

Disliker dislikes :-1:

Comment (lucas-tortora) [inline documentation/docs/decentralized_identity.md:34]: I think you may have accidentally deleted a sentence here

File: documentation/docs/decentralized_identity.md [hunk 2]
```diff
@@ -1,99 +1,103 @@
...
 
-IOTA Identity presents an opportunity for companies to embrace a new role in the ecosystem. Traditional approaches do not provide cost efficient solutions to new legislation like GDPR. IOTA Identity enables organizations to change their processes to comply with the new regulations in a cost efficient and privacy-enabling manner. Features of “Data Protection and Privacy by Design” shift responsibility over Personal Identifiable Information (PII) from organization to customer, and organizations no longer need to store that data. The relationship between customer and organization is also tightened as communication via a third party Identity provider like Google or Facebook is no longer needed.
+Many corporations are associated with greed and abuse of power. This reputation stems from the role some have chosen to take within society. Corporations are trusted with our data, but often do not act responsibly; vulnerability, fix, patch, repeat. In software and systems, we have seen this cycle repeat. Headlines on data leaks are now an ever-present feature in the news.
```
Comment (aleksei-korolev) [inline documentation/docs/decentralized_identity.md:48]: Can we be even loudmouther here?

```suggestion
Corporations are associated with greed and abuse of power. This reputation stems from the role some have chosen to take within society. Corporations are trusted with our data, but often do not act responsibly; vulnerability, fix, patch, repeat. In software and systems, we have seen this cycle repeat. Headlines on data leaks are now an ever-present feature in the news.
```

File: documentation/docs/decentralized_identity.md [hunk 3]
```diff
@@ -1,99 +1,103 @@
...
 
-Data must be immutably stored on a distributed ledger to ensure the decentralized nature of the digital identity protocol. With the network’s continual uptime, credentials are always verifiable without a dependency on the servers of credential issuers. This system also increases individual privacy, because contact with the Issuer is removed from the interaction between Holder and Verifier. Issuers will not be able to track when and how often the Holder uses their credentials. The flexibility afforded of the Tangle means that the digital identity framework remains extendable in the future.
+Data must be immutably stored on a distributed ledger to ensure the decentralized nature of the digital identity protocol. With the network’s continual uptime, credentials are always verifiable without a dependency on the servers of credential issuers. This system also increases individual privacy, because contact with the Issuer is removed from the interaction between Holder and Verifier. Issuers will not be able to track when and how often the Holder uses their credentials. The flexibility afforded from the Tangle means that the digital identity framework remains extendable in the future.
```
Comment (aleksei-korolev) [inline documentation/docs/decentralized_identity.md:98]: > The flexibility afforded from the Tangle

TIL that afford has a second meaning.

File: documentation/docs/getting_started/overview.md [hunk 4]
```diff
@@ -7,13 +7,14 @@ keywords:
...
 
-However, you can't tell much about the subject from a DID. You need to combine the DID with [Verifiable Credentials](../verifiable_credentials/overview). Verifiable Credentials are statements about the creator of the DID. They can be shared and verified online in a "Bring Your Own Identity" (BYOI) manner, and the DID creator remains in complete control of the process.
+However, you can not tell much about the subject from a DID. You need to combine the DID with [Verifiable Credentials](https://wiki.iota.org/identity.rs/verifiable_credentials/overview). Verifiable Credentials are statements about the creator of the DID. They can be shared and verified online in a "Bring Your Own Identity" (BYOI) manner, and the DID creator remains in complete control of the process.
```
Comment (aleksei-korolev) [inline documentation/docs/getting_started/overview.md:17]:
```suggestion
However, you cannot tell much about the subject from a DID. You need to combine the DID with [Verifiable Credentials](https://wiki.iota.org/identity.rs/verifiable_credentials/overview). Verifiable Credentials are statements about the creator of the DID. They can be shared and verified online in a "Bring Your Own Identity" (BYOI) manner, and the DID creator remains in complete control of the process.
```

**Review by aleksei-korolev — CHANGES_REQUESTED** [review]

File: documentation/docs/decentralized_identity.md (hunk 1 above)
Comment (charlesthompson3) [reply documentation/docs/decentralized_identity.md:34]: Thats...weird? I wouldn't have nuked a whole sentence. But noted.

File: documentation/docs/decentralized_identity.md [hunk 5]
```diff
@@ -1,99 +1,103 @@
...
 
-With digital identity, the user can decide what information to share and with whom they would like to share it. This will maintain and even improve people’s online privacy, while allowing many new features and new business opportunities. Businesses will be able to trust BYOI information, enhancing interactions between company and customer, but also reducing fraudulent cases and endemic security risks.
+With digital identity, the user can decide what information to share and with whom they would like to share it. This will maintain and even improve online privacy for people, while allowing many new features and new business opportunities. Businesses will be able to trust BYOI information, enhancing interactions between company and customer, but also reducing fraudulent cases and endemic security risks.
```
[the remark this answers is not in the document]
Comment (charlesthompson3) [reply documentation/docs/decentralized_identity.md:20]: 👍

File: documentation/docs/decentralized_identity.md [hunk 6]
```diff
@@ -1,99 +1,103 @@
...
 
 <iframe width="640" height="360" src="https://www.youtube.com/embed/4YnGFHhxua8" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 ## Identity for People
 
-:::info New internet
+:::note
```
[the remark this answers is not in the document]
Comment (charlesthompson3) [reply documentation/docs/decentralized_identity.md:26]: This is kind of a "pitch" article so I allowed it. Thoughts?

File: documentation/docs/decentralized_identity.md (hunk 2 above)
Comment (charlesthompson3) [reply documentation/docs/decentralized_identity.md:48]: Without getting preachy? Kind of strikes the right tone for me. @lucas-tortora ?

File: documentation/docs/decentralized_identity.md (hunk 3 above)
Comment (charlesthompson3) [reply documentation/docs/decentralized_identity.md:98]: 🤷👌

**Review by lucas-tortora — APPROVED** [review]

**Review by aleksei-korolev — APPROVED** [review]

**Comment by JelleMillenaar** [comment]
Unsigned commits approved, since they don't touch code. Have instructed contributors to sign commits in the future.

**Pull request merged**